File: src/main/java/no/nav/melosys/eessi/models/sed/nav/MedlemskapA008Bruker.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@SuppressWarnings("unused")
@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class MedlemskapA008Bruker {

    private ArbeidIFlereLand arbeidiflereland;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/MedlemskapA008Bruker.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class MedlemskapA008Bruker(
    // Any? fordi CDM 4.3 sender ArbeidIFlereLand (objekt) mens CDM 4.4 sender List<ArbeidIFlereLand> (array).
    // Ved deserialisering blir dette LinkedHashMap (objekt) eller List<LinkedHashMap> (array).
    var arbeidiflereland: Any? = null
)
